### DISCUSSION TITLE
How to code a global and generic CRUD api?

### DISCUSSION THREAD

**Discussion by davidgfr** [fastapi/fastapi#8652] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI
```

### Description

Hello,
We would like to have one api that can be generic and global for all the crud operations on the project. I mean we need to use the same api to use crud operations in different tables/databases with different fields,...
In the parameters of the called api, we should give the table name, the fields concerned and the appropriate crud method.
With that way, instead of having hundreds of crud apis, we will deal with only one api for all the crud operations.
Is there any idea on how this api can be done in fastapi (fyi, it is possible using php/symfony :( but I prefer find a solution using python/fastapi).
Thank you a lot for your understanding and your help.

### Operating System

Linux

### FastAPI Version

0.65.1

### Python Version

3.8.5

**Comment by insomnes** [comment]
So, what stops you in this case from sending table name and mtehod as lets say path parameters and table fields with may be query parameters.
It would be unsupportable mess, i think, but you will have only one handler, if you wish so.

**Comment by davidgfr** [comment]
Thank you for your answer.
In this project, I will have a lot of tables and probably hundreds of crud apis :(. The idea is to have to call only one api to manage all the crud operations by choosing the aimed db and table, the required fields and the crud method.
As I am a beginner in fastapi, I don't know if I can have such api and I would like that someone helps me to do that ;)

**Comment by duz-sg** [comment]
There is something similar implemented at:
https://github.com/tiangolo/full-stack-fastapi-postgresql/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/backend/app/app/crud/base.py
